### PR TITLE
Feature/421-contract agreement admin view

### PR DESCRIPTION
**Pull request by ds-mmaul**
resolves eclipse-tractusx/traceability-foss#

### CODE REVIEW

File: frontend/cypress/integration/pages/AdminContractView.ts [hunk 1]
```diff
@@ -0,0 +1,50 @@
+import { Then, When } from '@badeball/cypress-cucumber-preprocessor';
+
+
+export class ContractAdminView {
+
+}
+
+let currentContractId: string;
+
+When("navigate to view section {string} in administration view", (viewSection: string) => {
```
Comment (ds-mwesener) [inline frontend/cypress/integration/pages/AdminContractView.ts:10]: Can we use enums here?

**Review by ds-mwesener — CHANGES_REQUESTED** [review]
Please check if we can use enums within cypress tests.

File: frontend/cypress/integration/pages/AdminContractView.ts (hunk 1 above)
Comment (ds-mmaul) [reply frontend/cypress/integration/pages/AdminContractView.ts:10]: It seems that enums dont work, tested it out

Comment (ds-mmaul) [reply frontend/cypress/integration/pages/AdminContractView.ts]: Fixed the issue, enums working

**Pull request merged**